style.css

```diff
@@ -0,0 +1,44 @@
+#main
+    {
+      background-color:whitesmoke;
+    }
+
+#title
+    {
+       padding-left: 600px;
+       font-size: 40px;
+       font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    }
+    p{
+        padding-left: 575px;
+        font-size: 30px;
+        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    }
+#img-caption
+    {
+        padding-left: 400px;
+        font-size: 20px;
+        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    }
+#img-div
+    {
+        background-color:#77889938;
+        
+    }
+#img
```

Comment:
**Required Changes ♻️**

Check the comments under the review.

- [ ]  Kindly ensure to use Linters as it's a very accurate tool you can use to format your code perfectly.